docs: Event streaming documentation (v13) (#626)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/automation/event_streaming.md

2. Go to: **Home > Automation > Event Streaming > Event Consumer**.
3. Once you open the Event Consumer document you will see all the Document Types that the consumer has subscribed to. Change the status from 'Pending' to 'Approved' for all the Document Types that you want to approve to be consumed. You can change the status to 'Rejected' if you do not want the documents of that Document Type to be consumed.
4. Save.

    ![Event Consumer](/docs/assets/img/automation/event-consumer-doc.png)

>**Note**: Document updates for Subscribed Document Types won't be synced unless they are Approved.

### 2.3 Offline access with single site
If you have some places where internet connectivity is low, for example, a store in a remote area where sales invoices are generated and you want to sync these invoices from the store to the hosted account, you could setup offline syncing using the following steps:

> 1. Set up an ERPNext local instance. You could use this [guide](https://github.com/frappe/bench) for the same:

Reviewer: ```suggestion
1. Set up an ERPNext local instance. You can refer [this guide](https://github.com/frappe/bench) for local setup.
```

> 2. You will already have a hosted account with your company set up.

Reviewer: ```suggestion
2. You need to have hosted account with your company set up.
```
